packages/picasso/src/QuillEditor/formats/bold.ts

```diff
@@ -5,28 +5,23 @@ import getTypographyClassName from '../../Typography/utils/get-typography-class-
 
 const QuillBold = Quill.import('formats/bold')
 
-class TypographyBoldBlot extends QuillBold {
-  static typographyClasses: Classes
-  static create() {
-    const node = super.create()
-
-    node.setAttribute(
-      'class',
-      getTypographyClassName(this.typographyClasses, {
-        variant: 'body',
-        size: 'inherit',
-        weight: 'semibold',
-      })
-    )
-
-    return node
+const makeBoldFormat = (typographyClasses: Classes) =>
+  class LinkBlot extends QuillBold {
+    static create(value: string) {
+      const node = super.create(value)
+
+      node.classList.add(
+        getTypographyClassName(typographyClasses, {
+          variant: 'body',
+          size: 'inherit',
+          weight: 'semibold',
+        })
+          .split(' ')
+          .join(',')
+      )
```

Comment:
I don't think joining them with commas works as expected here, its adding a single CSS class with all the typographic classes joined by `,`. You can see on [this sandbox](https://codesandbox.io/s/nice-voice-hs5reh?file=/src/index.js) that `#app` gets a `foo,bar` CSS class

![image](https://user-images.githubusercontent.com/5845160/175941495-5303ab1d-9e07-4cc9-b533-0c456f51b597.png)

The function accepts n-arguments that would be the class names, so you can use the spread operator here:
```suggestion
      node.classList.add(
        ...getTypographyClassName(typographyClasses, {
          variant: 'body',
          size: 'inherit',
          weight: 'semibold',
        })
        	.split(' ')
      )
```

This is also valid for the other occurrences of `nodeList.add()` in the code

Reply:
Good catch. As `add` doesn't accept spaces I was going to split it but I accidentally joined them back with `,`

Reply:
Addressed :)